grafana/resource_synthetic_monitoring_probe_test.go

```diff
@@ -1,3 +1,4 @@
+//go:build cloud
```

Comment:
Do we really need this change for this PR to work?

Reply:
no, thanks for noticing. That's probably a side-effect of me opening the file looking for a place to add the tests.